app/models/vm_reconfigure_task.rb

```diff
@@ -25,7 +25,7 @@ def self.get_description(req)
     msg << build_message(options, :network_adapter_edit, "Edit Network Adapters: %d", :length)
     msg << build_message(options, :cdrom_connect, "Attach CD/DVDs: %d", :length)
     msg << build_message(options, :cdrom_disconnect, "Detach CD/DVDs: %d", :length)
-    "#{request_class::TASK_DESCRIPTION} for: #{resource_name(req)} - #{msg.compact.join(", ")}"
+    N_("#{request_class::TASK_DESCRIPTION} for: #{resource_name(req)} - #{msg.compact.join(", ")}")
```

Comment:
This one is a bit more complicated, but it's a similar pattern - you can build up the list of messages and each one should be translated before interpolating it in.